src/lib/rules/install-dependencies.ts

```diff
@@ -0,0 +1,26 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { PackageJson } from '../package-json';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+export interface InstallOptions {
```

Comment:
Can you extract the interface into an own file?

Reply:
I already feared that ;-P